crates/errors/src/error_codes/META0014.md

```diff
@@ -0,0 +1,11 @@
+## META0014
+
+Service discovery response failed, and the server may have responded in HTTP1.1.
+This can happen when discovering locally running dev servers from Faas platforms
+eg `wrangler dev` or `cargo lambda watch`. FaaS platforms in production
+generally will support HTTP2, however, so this is generally only a local
+development concern.
+
+You can try to discover the endpoint with `use_http_11` set to true when working
```

Comment:
is it enough to pass the flag like `--use_http_11` or do I have to pass `true` as well?

Reply:
it is enough ✅